packages/core/admin/_internal/cli/commands/build.ts

```diff
@@ -21,8 +23,31 @@ const build = async (options: BuildCLIOptions) => {
       );
     }
 
+    if (options.bundler !== 'webpack') {
+      options.logger.log(
+        boxen(
+          `Using ${chalk.bold(
+            chalk.underline(options.bundler)
+          )} as a bundler is considered experimental, use at your own risk. If you do experience bugs, open a new issue on Github – https://github.com/strapi/strapi/issues/new?template=BUG_REPORT.md`,
+          {
+            title: 'Warning',
+            padding: 1,
+            margin: 1,
+            align: 'center',
+            borderColor: 'yellow',
+            borderStyle: 'bold',
+          }
+        )
+      );
+    }
+
     const envSourceMaps = process.env.STRAPI_ENFORCE_SOURCEMAPS === 'true';
 
+    /**
+     * ENFORCE NODE_ENV to production when building
+     */
+    process.env.NODE_ENV = process.env.NODE_ENV ?? 'production';
```

Comment:
Should
```
run yarn build --bundler=vite
run yarn start
```

always run strapi in production mode?

That's the behaviour for me even when NODE_ENV is set as 'development'

Reply:
I think it should be yeah, we have `develop` for a reason? The issue of not forcing `production` mode in Vite is a whole host of issues to do with JSX & React node_modules which was a massive headache. Do you see a downside? 🤔

Reply:
No that makes sense to me. I'm a bit confused about how this always defaults to 'production' though, if I have NODE_ENV as 'development' in .env I'd expect it to use that ?

Reply:
oh i see, sorry i misunderstood. I'll look at that :)